generative/maisi/configs/environment.json

```diff
@@ -0,0 +1,14 @@
+{
+    "model_dir": "/mnt/drive1/canz/models/",
```

Comment:
Hi @Can-Zhao , can we remove personal info (`canz` in the file path) ? Thanks!

Reply:
Just FYI, most tutorials use `MONAI_DATA_DIRECTORY` to hide similar path issues

https://github.com/Project-MONAI/tutorials/blob/main/CONTRIBUTING.md#common-recommendations-for-the-review

Reply:
Thanks! Done